### ISSUE TITLE
[What do you think?] - Number of tickets awaiting to be served

### ISSUE THREAD

**Issue by ldardini** [boxbilling/boxbilling#424]
Hello, I have a patch coded in my version showing to the client the number of tickets awaiting to be server in the same helpdesk where they open the ticket. Do you think it will be accepted in the main branch? 

![image](https://cloud.githubusercontent.com/assets/1225823/12529657/e5ff949a-c1bf-11e5-888c-46e9f456354f.png)

**Comment by ldardini** [comment]
I have uploaded a pull request with this code. It is code I have paid a company to develop. I have reviewed it and I grant for it.

**Comment by Mecanik** [comment]
I am SO, SO dissapointed in BoxBilling. I have changed serious providers because the indian guys from boxbilling said my CRON is not working because of the hosting provider, but still no solution.
I have paid for almost 2 years every month for the subscription, and I don`t get anykind of help.
Not to mention a few days ago, one of my customers added 250$ in his account, and I had to remove the ballance and invoices, I reported to boxbilling, and again, the indian guy told me there is nothing wrong, nothing happened.
If I could have just open my eyes really big when I installed this crap... Now I can`t even move to WHMCS, because I sold software with Licenses generated by boxbilling.
I just want to say, there is nothing working properly, and I mean nothing. I could just fix the issues, because I myself am a developer, but why ? why if I am paying these guys from the beginning ?
Horrible experience, HORRIBLE. Don`t waste time on developing on this shit, if some of you want, I own a software company, and we can build a billing system from the scratch. At least a working one... not fucking up people`s business.
